packages/core/helper-plugin/src/index.js

```diff
@@ -102,6 +102,7 @@ export { default as prefixPluginTranslations } from './utils/prefixPluginTransla
 export { default as pxToRem } from './utils/pxToRem';
 export { default as request } from './utils/request';
 export { default as setHexOpacity } from './utils/setHexOpacity';
+export { getDisplayName } from './utils/getDisplayName';
```

Comment:
What do you think about not sharing this utils from the helper-plugin for now, since you only need it in the CM?

This would give us some time to figure out how we want this to work so that we can apply it everywhere.

Reply:
It could be valuable to plugin developers but I tend to agree with Gustav and leave it in the CM for now if that's currently the only place it is used. Fernando is on vacation this week but he might have an opinion as well.

Reply:
If this is the last open discussion I'd prefer to make the util local to the CM and move on. We can always expose more, but we can never remove something publicly exposed.

Reply:
will do 👍 sorry I have been busy with other topics today, I will move the util into CM